src/fio/fioEngine.js

```diff
@@ -218,10 +205,13 @@ export class FioEngine extends CurrencyEngine {
           apiUrl,
           this.fetchCors
         )
-        if (actionName === 'getChainInfo') {
-          return fioSDK.transactions.getChainInfo()
+
+        switch (actionName) {
+          case 'getChainInfo':
+            return fioSDK.transactions.getChainInfo()
```

Comment:
If this access the network then it should use multicastServers

Reply:
yes, but this is in the multicastServers method already